app/component/map/tile-layer/VectorTileLayerContainer.js

```diff
@@ -26,8 +27,12 @@ export default function VectorTileLayerContainer(props, { config }) {
       layers.push(TicketSales);
     }
 
-    if (config.dynamicParkingLots && config.dynamicParkingLots.showDynamicParkingLots) {
-      layers.push(DynamicParkingLots)
+    if (config.constructor && config.dynamicParkingLots.showDynamicParkingLots) {
+      layers.push(DynamicParkingLots);
+    }
+
+    if (config.constructor && config.roadworks.showRoadworks) {
```

Comment:
Above, the existence of config.<layer> was checked before accessing the show<Layer> property. Could you explain why config.constructor makes this unnecessary? What would happen for e.g. a HSL config if it roadworks is not defined at all?

Reply:
To be honest, I have no idea why I put this here. I suspect a copy'n paste from somewhere else.

I changed it.